libsnark/zk_proof_systems/plonk/tests/test_plonk.cpp

```diff
@@ -538,24 +386,23 @@ template<typename ppT, class transcript_hasher> void test_plonk_prover_rounds()
     Field secret = example.secret;
     // example witness
     std::vector<Field> witness = example.witness;
-    // example circuit
-    circuit_t<ppT> circuit = plonk_circuit_description_from_example<ppT>(
-        example.gates_matrix_transpose,
-        example.wire_permutation,
-        example.public_input,
-        example.public_input_index);
+
     // hard-coded values for the "random" blinding constants from
     // example circuit
     std::vector<libff::Fr<ppT>> blind_scalars = example.prover_blind_scalars;
     // maximum degree of the encoded monomials in the usrs
     size_t max_degree = PLONK_MAX_DEGREE;
 
     std::shared_ptr<libfqfft::evaluation_domain<Field>> domain =
-        libfqfft::get_evaluation_domain<Field>(circuit.num_gates);
+        libfqfft::get_evaluation_domain<Field>(example.num_gates);
 
     // prepare srs
     usrs<ppT> usrs = plonk_usrs_derive_from_secret<ppT>(secret, max_degree);
-    srs<ppT> srs = plonk_srs_derive_from_usrs<ppT>(usrs, circuit);
+    srs<ppT> srs = plonk_srs_derive_from_usrs<ppT>(
+        usrs,
+        example.gates_matrix,
+        example.wire_permutation,
+        example.PI_wire_index);
```

Comment:
"indices" - even though there is 1, this should be a vector.

Reply:
Fixed.